services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -255,7 +256,6 @@ export const SubmissionSummary = (): React.ReactElement => {
 
     const onModalSubmit = async (values: typeof modalFormInitialValues) => {
         const { unlockReason } = values
-        modalRef.current?.toggleModal(undefined, false)
```

Comment:
This was a bug fix for something @rswerve found which was if you dont close the modal immediately its possible for someone to unlock twice and just interact with the form more while submit is in progress. 

Rather than keep the modal open longer here to fix delay to the redirect we could do something like show the loading page instead while things are delayed for an async request